server/base/src/main/java/org/apache/accumulo/server/fs/VolumeChooser.java

```diff
@@ -28,5 +29,5 @@
  * benefit from using per-table configuration using {@link Property#TABLE_ARBITRARY_PROP_PREFIX}.
  */
 public interface VolumeChooser {
-  String choose(VolumeChooserEnvironment env, String[] options);
+  String choose(VolumeChooserEnvironment env, String[] options) throws AccumuloException;
```

Comment:
AccumuloException is really a client-side exception, and probably shouldn't be used internally on the server-side.